quiz-bundle-spec.md

```diff
@@ -0,0 +1,71 @@
+# Qwiklabs Quiz Bundle Specification
+
+**Version 1**
+
+> This is a DRAFT document. We welcome feedback as this format evolves.
```

Comment:
I know you must have copied this from another file, but it's never too late to bring up weirdness.

When this gets merged to master, is this file really a draft? Sure, there may be some tweaks as you write the actual implementation, but this is going to be sorta-kinda codified as V1 when the website implementation unbundles these in Dec/Jan.

Reply:
I personally think we should remove these versions and draft messages on all files ASAP. We're already writing unbundling code for game templates, so these are definitely real, not draft, specs.

In the meantime I'm happy to remove it here - or just wait and do all specs at once in a follow up PR for consistency.

Reply:
Just because there exists an implementation of a spec, does not mean the spec if finalized. Even after Lab bundles were supported by GitWhispere/Website, they were still drafts. We have not guaranteed that any of the features described in this document will be supported by external systems.

Heck, the bundling/unbundling code we implemented did *not* match the Lab bundle spec described in this repo until very recently.

Now that....

- the spec has been updated to reflect reality of our implementation, and
- we've got multiple engineers contributing to this and the unbundling code has gone through a couple revisions...

... I'm willing to remove the V1 draft status at the end of this quarter.

Reply:
@supersam654 and @rickcnagy  ^^

Reply:
@jgan EOQ1 sounds great 😄